lua/entities/gmod_wire_expression2/init.lua

```diff
@@ -601,8 +602,18 @@ function ENT:Reset()
 	-- prevent E2 from executing anything
 	self.context.resetting = true
 
+	if self.context.timebench > e2_timequota then -- Abusing reset() at end of excution.
+		-- E2Lib.abuse(self.player)
```

Comment:
you left a comment in here

Reply:
Yea left it there for people to comment on whether it should exist at all. But that time check shouldn't exist now that divran reminded me it's supposed to be disabled by default for being screwed up.